user/browserstack.md

```diff
@@ -4,7 +4,7 @@ layout: en
 
 ---
 
-Travis CI integrates with [BrowserStack](https://www.browserstack.com), a cross browser and real device
+Travis CI integrates with [BrowserStack](https://www.browserstack.com/?utm_source=rapise&utm_medium=partnered), a cross browser and real device
```

Comment:
Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered